scripts/test/lint.py

```diff
@@ -304,7 +304,6 @@ class UsesInvalidCategory(Lint):
     EXCLUSIONS = [
         ".ollydumpex.vm",
         ".scyllahide.vm",
-        "7zip-15-05.vm",
```

Comment:
[nitpick] we need to remove idafree and microsoft-windows-terminal from the list. @emtuls do you want to do it as part of this PR?